RELEASE.md

```diff
@@ -258,13 +267,15 @@ Then, the Release Manager releases and announces the new version:
          git checkout x.x.x
          ```
 
-    3.  Tag the commit with the current release version prefixed with `self-hosted-`:
+    3.  For the CLI Coverage Reporter, you must update the `SELF_HOSTED_CODACY_REPORTER_VERSION` flag on the `get.sh` script found in the project's root. This should reflect the new version that is intended to be used in the new self-hosted release. You do not have to do this for the Analysis CLI tool.
```

Comment:
Tweak for conciseness

```suggestion
    3.  For the CLI Coverage Reporter, you must update the `SELF_HOSTED_CODACY_REPORTER_VERSION` flag on the `get.sh` script found in the project's root. This should match the version used by the new self-hosted release. You do not have to do this for the Analysis CLI tool.
```

Reply:
As above, let's use the "official" names:

```suggestion
    3.  For the Codacy Coverage Reporter, you must update the `SELF_HOSTED_CODACY_REPORTER_VERSION` flag on the `get.sh` script found in the project's root. This should reflect the new version that is intended to be used in the new self-hosted release. You do not have to do this for the Codacy Analysis CLI.
```